components/StatusDropdown/styles.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FilterDropdownStyles = styled.div`
+  position: relative;
+  background: ${COLORS.white};
+  width: 10.5rem;
+  border-radius: 0.5rem;
+  padding-bottom: 0.5rem;
+`;
+
```

Comment:
Make a new styled div. This is so we can house all the content in one place and add the padding around it (so that if Josh ever changes it again it's easy to change the padding directly from the figma). Use the styled div to wrap around the content (check my comments in index.tsx)
```
export const FilterContentDiv = styled.div`
  display: flex;
  flex-direction: column;
  padding: 0.625rem 1.25rem 0.9rem 1.25rem;
  align-content: space-between;
`;
```